html/webappapis/dynamic-markup-insertion/opening-the-input-stream/resources/history-frame.html

```diff
@@ -0,0 +1,15 @@
+<script>
+function queueTest() {
+  // The timeout is necessary to avoid the parser still being active when
+  // `document.open()` is called and becoming a no-op.
+  setTimeout(() => {
```

Comment:
You should be able to grab opener.t.step_timeout or something like that. That would help catch any exceptions, and allow you to avoid the lint whitelist.

Reply:
I'm reluctant to do that, as it will cause the entry settings object to be that of the opener's, which has the potential of opening a can of worms.

Reply:
Blehhh. Can you include testharness.js and use the global step_timeout?